packages/Select/src/select.scss

```diff
@@ -1,93 +1,142 @@
+    --ids-select-border-radius: var(--hop-shape-rounded-md);
+    --ids-select-broder-width: 0.0625rem;
+
+    /* Compact */
+    --ids-select-font-size-compact: var(--hop-body-sm-font-size);
+    --ids-select-line-height-compact: var(--hop-body-sm-line-height);
+
+    /* Hover */
+    --ids-select-background-hover: var(--hop-neutral-surface-hover);
+    --ids-select-border-color-hover: var(--hop-neutral-border-hover);
+
+    /* Active */
+    --ids-select-border-color-active: var(--hop-neutral-border-active);
+
+    /* Focus */
+    --ids-select-background-focus:  var(--hop-neutral-surface-hover);
+    --ids-select-border-color-focus: var(--hop-neutral-border-hover);
+    --ids-select-focus: 0 0 0 0.12rem var(--hop-neutral-surface), 0 0 0 0.219rem var(--hop-primary-border-focus);;
+
+    /* Disabled */
+    --ids-select-color-disabled: var(--hop-neutral-text-disabled);
+    --ids-select-background-disabled: var(--hop-neutral-surface-disabled);
+    --ids-select-border-color-disabled: var(--hop-neutral-border-disabled);
+
+    /* Error */
+    --ids-select-border-color-error: var(--hop-danger-text-weak);
 }
 
 .ids-select {
-  color: var(--ids-select-color);
-  font-size: var(--ids-select-font-size);
-  font-family: var(--ids-select-font-family);
-  font-weight: var(--ids-select-font-weight);
-  line-height: var(--ids-select-line-height);
-  width: 100%;
-  padding: 0 var(--ids-select-padding);
-  position: relative;
-  background-color: var(--ids-select-background);
-  border: .1rem solid var(--ids-select-border-color);
-  border-radius: var(--ids-select-border-radius);
-  box-sizing: border-box;
-  transition: border-color .2s ease 0s;
-  cursor: pointer;
-  user-select: none;
-
-  &,
-  & * {
+    color: var(--ids-select-color);
+    font-size: var(--ids-select-font-size);
+    font-family: var(--ids-select-font-family);
+    font-weight: var(--ids-select-font-weight);
+    line-height: var(--ids-select-line-height);
+    width: 100%;
+    padding: 0;
+    position: relative;
+    background-color: var(--ids-select-background);
+    border: var(--ids-select-broder-width) solid var(--ids-select-border-color);
+    border-radius: var(--ids-select-border-radius);
     box-sizing: border-box;
-  }
-
-  &:hover {
-    border-color: var(--ids-select-border-color-hover);
-  }
-
-  &:focus,
-  &.focus {
-    outline: 0;
-    box-shadow: none;
-    border-color: var(--ids-select-border-color-active);
-  }
-
-  &--compact {
-    font-size: var(--ids-select-font-size-compact);
-    line-height: var(--ids-select-line-height-compact);
-  }
-
-  &--active,
-  &.active {
-    border-color: var(--ids-select-border-color-active);
-  }
-
-  &--disabled,
-  &--disabled:focus {
-    color: var(--ids-select-color-disabled);
-    background-color: var(--ids-select-background-disabled);
-    border-color: var(--ids-select-border-color-disabled);
-    cursor: not-allowed;
+    transition: border-color .2s ease 0s;
```

Comment:
whats the use of the 0s here?

Reply:
it's not a change I've made, I don't know the reason for such timing